openscm/definitions/__init__.py

```diff
@@ -0,0 +1,9 @@
+from os.path import join, dirname
+
+
+import pint
+
+unit_registry = pint.UnitRegistry()  # start a unit repository using the default variables
+# should move the definitions somewhere more stable
+# openscm/definitions ?
+unit_registry.load_definitions(join(dirname(__file__), 'emissions_units.txt'))
```

Comment:
Can we avoid parsing a file and define the units in-code?

Reply:
It's possible to do these definitions programmatically, but not the library itself uses textfiles.

Reply:
Hmm, I hate parsing text files, but ok ;)